res/firmware.nsh

```diff
@@ -138,6 +138,13 @@ if "%2" == "bios" then
             exit %lasterror%
         endif
 
+        # Flash with ASRockAfuEfix64 and exit if possible
+        # For: thelio-spark-b1
+        if exist ASRockAfuEfix64.efi then
+            ASRockAfuEfix64.efi firmware.rom /P /B /N /K /X /REBOOT
```

Comment:
Usually we don't want to reboot from the flashing command

Reply:
I'll do a `/SHUTDOWN` instead then

Reply:
We don't want to do that either. Ideally the flashing command exits cleanly and returns control to the flashing program. Otherwise, you need to make a new tag.

Reply:
Gotcha, that makes sense. I've removed `/SHUTDOWN` as well.